src/yaml/ingestionTrigger/metadata/rasterLayerMetadata.yaml

```diff
@@ -0,0 +1,71 @@
+openapi: "3.0.1"
+info:
+  title: layer metadata definitions
+  version: 1.0.0
+components:
+  schemas:
+    updateRasterLayerMetadata:
+      type: object
+      required:
+        - region
+        - classification
+      description: layer meta data that will override
+      properties:
+        productSubType:
+          type: string
+          description: Layer's sub type
+        description:
+          type: string
+          description: layer's description
+        region:
+          type: string
+          description: region of record
+        classification:
+          type: string
```

Comment:
should be defined as enum with the values defined in meta-data file

Reply:
reverted